src/modules/organisation/api/organisation.controller.spec.ts

```diff
@@ -261,41 +261,52 @@ describe('OrganisationController', () => {
         const params: OrganisationByIdParams = {
             organisationId: faker.string.uuid(),
         };
+        describe('when usecase returns a OrganisationResponse', () => {
```

Comment:
I know we're not totally consistent, but a few empty lines would aid readability (also in line 301)

```suggestion
        };

        describe('when usecase returns a OrganisationResponse', () => {
```

Reply:
fixed